BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -510,39 +534,48 @@ extension ConvivaAnalytics: BitmovinPlayerListenerDelegate {
 
     // MARK: - Ad events
     func onAdStarted(_ event: AdStartedEvent) {
+        var adInfo = event.ad.adInfo
         let adPosition: ConvivaSDK.AdPosition = AdEventUtil.parseAdPosition(
             event: event,
             contentDuration: player.duration
         )
-        var adAttributes = [String: Any]()
-        adAttributes["c3.ad.position"] = adPosition.rawValue
-        videoAnalytics.reportAdBreakStarted(
-            AdPlayer.ADPLAYER_CONTENT,
-            adType: AdTechnology.CLIENT_SIDE,
-            adBreakInfo: adAttributes
+        adInfo["c3.ad.position"] = adPosition.rawValue
+
+        adAnalytics.reportAdLoaded(adInfo)
+        adAnalytics.reportAdStarted(adInfo)
+        adAnalytics.reportAdMetric(
+            CIS_SSDK_PLAYBACK_METRIC_PLAYER_STATE,
+            value: PlayerState.CONVIVA_PLAYING.rawValue
         )
     }
 
     func onAdFinished() {
-        videoAnalytics.reportAdBreakEnded()
+        adAnalytics.reportAdEnded()
```

Comment:
This change is a behaviour change of the integration!!! (and of course the related changes i.e. moving the orignal call into the ad break finished event, and the starting calls in the ad started and ad break started)